whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/DummifierNrtm.java

```diff
@@ -236,6 +236,27 @@ public static RpslObject getPlaceholderPersonObject() {
         );
     }
 
+    public static RpslObject getPlaceholderPersonObject(final CIString source){
```

Comment:
This new method is almost identical to the existing `getPlaceholderPersonObject` in that class. Refactor the original method by calling `getPlaceHolderPersonObject(CIString.ciString("RIPE"))` to avoid duplicating the object, or replace the original method altogether.

Reply:
yes, done